system-int-test/test/cmr/system_int_test/search/generic_association_test.clj

```diff
@@ -248,9 +250,10 @@
         (is (= [tl1-concept-id]
                (:associations grid-search-result)
                (:associations grid-search-result-update)))
-        (is (= [{:concept-id tl1-concept-id :revision-id tl1-revision-id}]
-               (:association-details grid-search-result)
-               (:association-details grid-search-result-update)))
+        (is (= [{:concept_id tl1-concept-id :revision_id tl1-revision-id}]
+        ;; The grid is being searched from the .json endpoint which uses snake case assocation-details
```

Comment:
check with the style guide, but I think this needs to be aligned with code below.

Reply:
Adjusting comments for indentation